index.Rmd

```diff
@@ -0,0 +1,360 @@
+       x = "Year",
+       y = "Number of Pages") +
+  scale_x_continuous(limits = c(2010,NA), breaks = seq(2010,2022, by = 2)) +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.subtitle = element_text(size = 13),
+        plot.title = element_text(face = "bold"),
+        panel.grid.major.y=element_line())
+
+
+jmh_read_clean %>%
+  count(format) %>% 
+  mutate(format = fct_reorder(format, n)) %>% 
+  ggplot(aes(y = format,
+             x = n)) +
+  geom_col(fill = lgr_pink) +
+  labs(title = "Format Read",
+         subtitle = "For User: <jesshirsch26> ",
+       x = "",
+       y = "Book Format") +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.subtitle = element_text(size = 13),
+        plot.title = element_text(face = "bold"),
+        panel.grid.major.y=element_line())
+
+```
+
+The looking at an individuals reading data give a little more insight into where to focus. books read over time trends with page over time, which is not always the case. if the books read were a lot of graphic novels or audio books there would be a high number of books and a lower number of pages. this is why looking at the format can be important. when trying to make recommendation it's great to be able to tailor that towards a certain medium. in this case the spread is fairly diverse across physical, audio, and ebooks. reading tends are also tracking with world events, there is a slight uptick in 2016-17 and then a large up tick in 2020 during Covid (no wonder I started a blog).
+
+```{r echo=FALSE, message=FALSE, warning=FALSE}
+
+jmh_read_clean %>%
+  ggplot(aes(x = year_date_read,
+             y = read_time,
+             group = year_date_read)) +
+  geom_boxplot(fill = lgr_pink, color = "black") +
+  labs(title = "Average Time to Finish a Book",
+         subtitle = "For User: <jesshirsch26> ",
+       x = "Year",
+       y = "# of Days") +
+  coord_cartesian(ylim = c(0, 75),
+                  xlim = c(2010, 2022)) +
+  scale_x_continuous(breaks = c(2010, 2014, 2018, 2022)) +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.subtitle = element_text(size = 13),
+        plot.title = element_text(face = "bold"),
+        panel.grid.major.y=element_line())
+
+jmh_read_clean %>% 
+  group_by(year_date_read) %>% 
+  mutate(avg_pages = mean(number_of_pages, na.rm = T),
+         avg_time = mean(read_time, na.rm = T)) %>%
+  ungroup() %>%
+  ggplot(aes(x = year_date_read)) +
+  geom_col(aes(y = avg_pages/100), fill = lgr_grey) +
+  geom_line(aes(y = avg_time), color = lgr_yellow, size = 1.5) +
+  scale_x_continuous(limits = c(2010,NA), breaks = seq(2010,2022, by = 2)) +
+  scale_y_continuous(name = "Average Time to Read",
+                     sec.axis = sec_axis(trans = ~.*100, name = "Average Pages")) +
```

Comment:
This is a bit of personal preference, but I'd personally recommend against using a secondary axis. They're typically very difficult to interpret. In this case, I had to spend quite a bit of time reading your graph in order to figure out which axis corresponded with which geometry.

Perhaps a different approach might be to wrap this into a calculated field that relates `avg_time` to `avg_pages`?

```
jmh_read_clean %>% 
  group_by(year_date_read) %>% 
  mutate(avg_pages = mean(number_of_pages, na.rm = T),
         avg_time = mean(read_time, na.rm = T)) %>%
  mutate(time_per_page = avg_pages / avg_time) %>% 
  ungroup() %>%
  ggplot(aes(x = year_date_read,
             y = time_per_page)) +
  geom_line()
```

<img width="706" alt="image" src="https://user-images.githubusercontent.com/16887262/167529584-d4729e9f-9d38-4a94-b352-9608ab8fef93.png">